unoplat-code-confluence-commons/AGENTS.md

```
@@ -30,23 +30,23 @@ This project uses Backlog.md MCP for all task and project management. Before cre
 <!-- UNOPLAT_CODE_CONFLUENCE_CONTEXT:BEGIN -->
 <CRITICAL_INSTRUCTION>
 
-> Generated from branch `dev` at commit `5ecdba39d57f50c5188a8e32b9dd4f52d01611fe` (2026-07-18). Content may become stale as new commits land.
+> Generated from branch `dev` at commit `29472803c5188bd9b6c4aab5927632a203b7ca5d` (2026-08-05). Content may become stale as new commits land.
 
 </CRITICAL_INSTRUCTION>
 
 ## Engineering Workflow
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)